app/webserver/messages.go

```diff
@@ -73,7 +73,7 @@ type GetMessageListMessage struct {
 }
 type GetMoreMessages struct {
 	Type   string `json:"request"`
-	LastID string `json:"lastId"`
+	SentAt uint64 `json:"sentAt"`
```

Comment:
In `app/store/db.go`, sentat is specified as an integer. For me to learn, what makes uint64 the best option for this case?

Reply:
In `app/store/message.go` its unit64, which is (at least for me feals like it) a typical type for a timestamp.

For outgoing messages it is initialised with `uint64(time.Now().UnixNano() / 1000000)`, which would fit in a smaller type, I think.

In the database its `integer` because sqlite only has `integer` or `real` types for numbers. https://www.sqlite.org/datatype3.html

Reply:
I see! Thanks for the explanation!